vizro-core/src/vizro/managers/_data_manager.py

```diff
@@ -16,6 +16,26 @@
 pd_LazyDataFrame = Callable[[], pd.DataFrame]
 
 
+class VizroDataSet:
```

Comment:
```suggestion
class _Dataset:
```
Let's leave private for now (and maybe forever depending on how the API evolves) 🙂